corehq/apps/user_importer/importer.py

```diff
@@ -358,6 +358,8 @@ def create_or_update_users_and_groups(domain, user_specs, upload_user, group_mem
 
                     if is_password(password):
                         user.set_password(password)
+                        # overwrite password in results so we do not save it to the db
+                        status_row['row']['password'] = 'UPDATED'
```

Comment:
Nitpick: I've seen `******` used for this kind of thing and I think it's intuitive, what do you think?

Reply:
We currently ignore any sequence of `*`s uploaded in this field to allow reupload of a downloaded file since we default the password to that in the downloads. This allows you to distinguish between an upload that included literal `******` and therefore did not modify the password, and one that actually updates the password. Happy to change the constant, but I do not think `*****` would be a good value to use

Reply:
Cool, I didn't know that. How about `REDACTED`?

Reply:
oh wow, I didn't know we were already storing everything in the status_row.
If I am reading it correctly, we also return it back in the response? I don't think that should be needed at all, unnecessary response load and plus it has passwords, which i believe we are changing here now. but even then, we should not send it back

Reply:
We dont send this all back in the response, just certain info from it like counts and specific error messages if problems occurred. We store it in the celery results table, but only temporarily. This ensures the storage of results is more permanent.